e2e/features/registration.feature

```diff
@@ -1,10 +1,33 @@
 Feature: Registration
 
-  Scenario: Weak Password
+
+  Scenario Outline: Invalid email address
     Given the user is on the registration page
+    When the user enters an invalid "<emailAddress>" address (e.g., missing @)
```

Comment:
The scenario outline should not contain specifics/examples in the steps, you should rather have this as an input (as you do too).

Reply:
Im not sure what you mean. In the invalid email scenario the emailaddress is an input at the "when" step, as thats what Im testing and have examples of.

In the weak password scenario you also have the password as an input, so what should be changed here? 

Or do you mean that ALL input fields should be treated as an input in both scenarios and specify the constant values in the steps file as const variables?

Reply:
```suggestion
    When the user enters an invalid "<emailAddress>" address
```

I meant that the example at the end of the sentence (or "Step") is one of the examples, I would rather not create a matcher with it, since it is in the examples "array".